changelog.d/383.change.rst

```diff
@@ -0,0 +1,3 @@
+``in_()`` validators now raise a ValueError with a useful message
```

Comment:
It would be faster to fix it myself, but for the next time:

- We use [semantic newlines](http://rhodesmill.org/brandon/2012/one-sentence-per-line/).
- We use fully qualified names for functions (e.g. `attr.validators.in_()`) which simplifies the sentences – “`attr.validators.in_()` now raises…”.
- “raises an unhelpful TypeError” makes it sound like it’s present behavior which it’s not. :)
- Please wrap symbols like ValueError or TypeError in double ticks.

Yeah, sorry I care about changelogs. 😇

Reply:
No worries, I care about them too!  Fixed ❤️